CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 ## Unreleased
 
+### Fixes
+
+- Skips ignoring require cycle logs for RN > 0.70 ([#4214](https://github.com/getsentry/sentry-react-native/pull/4214))
```

Comment:
This may give the impression that it is not ignored on RN 0.70, so here is a wording suggestion:

```suggestion
- Skips ignoring require cycle logs for RN 0.70 or newer ([#4214](https://github.com/getsentry/sentry-react-native/pull/4214))
```